db/migrations/20230404074446_create_order_table.sql

```diff
@@ -0,0 +1,17 @@
+-- migrate:up
+CREATE TABLE orders {
+  id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  user_id INT NOT NULL,
+  total_price DECIMAL(8,2) NOT NULL,
+  name VARCHAR(500) NOT NULL,
```

Comment:
```suggestion
  name VARCHAR(500) NOT NULL,
```
name column은 어떤걸 위해 필요한 걸까요? order_number를 의미하는걸까요?

Reply:
주문 번호를 입력하려고 했던 건데 number로 변경하겠습니다.